### DISCUSSION TITLE
How does the agent know, which php app we want to monitor?

### DISCUSSION THREAD

**Discussion by ashu99099** [SkyAPM/SkyAPM-php-sdk#477] [Q&A] [answered]
How does the agent know, which php app we want to monitor? do we provide some info about the php app that we want to monitor?
Can we run the php app in docker?

**Comment by heyanlong** [comment, marked as answer]
https://github.com/SkyAPM/SkyAPM-php-sdk/blob/v4.1.2/docs/install.md
